README.md

```diff
@@ -35,6 +35,8 @@ packageProject(
   # (optional) create a header containing the version info
   # note that the path should be lowercase 
   VERSION_HEADER "${PROJECT_NAME}/version.h"
+  # (optional) install your library with a namespace (do NOT add extra '::')
+  LIBRARY_NAMESPACE ${MY_LIBRARY_NAMESPACE}
```

Comment:
For consistency with the other examples we could stick with 

```suggestion
  LIBRARY_NAMESPACE ${PROJECT_NAMESPACE}
```

Reply:
I will change that